ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -185,6 +185,10 @@ class JITModBuilder {
   // The LLVM type for a C `int` on the current machine.
   Type *IntTy;
 
+  // Map JIT instruction to basic block index and instruction index of the
+  // corresponding AOT instruction.
+  std::map<Value *, std::tuple<size_t, size_t, Instruction *>> AOTMap;
```

Comment:
What's the third element in the tuple? The comment mentions only two elements.

Reply:
Looks like the AOT instruction?

Reply:
Oh, that's the original AOT instruction. Will fix.